app/default/src/Elasticsearch/Repository/CarRepository.php

```diff
@@ -33,6 +36,11 @@ public function find(Pagination $pagination, ?CriteriaInterface $criteria = null
 
 		$response = $this->client->search($this->index, $query);
 
-		return $this->hydrator->hydrate($response);
+		$this->logger->info('Found: [' . count($response->results()) . ']' . ' for query: ' . json_encode($query->toArray()));
+
+		$hydra = $this->hydrator->hydrate($response);
+
+		// var_dump($hydra); die;
```

Comment:
@212223 😉 
```suggestion
```

Reply:
Thank you for spotting it. Done in [59e47a7](https://github.com/lrynek/painless-car-rental/pull/1/commits/59e47a75572e500cd7070e0f4740bb8ed76843cd)